homeassistant/components/climate/sensibo.py

```diff
@@ -191,6 +192,14 @@ def current_humidity(self):
         """Return the current humidity."""
         return self._measurements['humidity']
 
+    @property
+    def current_battery(self):
+        """Return the current battery voltage."""
+        if 'batteryVoltage' in self._measurements:
+            return self._measurements['batteryVoltage']
+        else:
+            return None
```

Comment:
Using `return self._measurements.get('batteryVoltage')` is enough.

Reply:
Thanks, I wasn't able to confirm whether this field is returned by the API for v2 pods (which don't have a battery) so I erred on the safe side.